lumicks/pylake/detail/bead_cropping.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+
+
+def find_beads(
+    summed_kymo, bead_diameter_pixels, plot=False, percentile=70, allow_negative_beads=True
+):
+    """Find bead edges in kymograph image sum
+
+    Attempts to search for the bead edges by checking where the fluorescence drops below a
+    threshold. Only intended for non-moving beads.
+
+    Parameters
+    ----------
+    data : np.ndarray
+        Kymograph image data (single channel summed along the time axis).
+    bead_diameter_pixels : float
+        Estimate for the bead size in pixels.
+    plot : bool
+        Plot result
+    percentile : int
+        Percentile below which to drop.
+    allow_negative_beads : bool
+        Tries to estimate a background from the dominant mode of the photon counts. This estimate
+        is subtracted from the raw counts prior to analysis.
+
+    Raises
+    ------
+    RuntimeError
+        When the algorithm fails to locate two edges during the bead finding stage.
+    """
```

Comment:
it might be nice to add a super short `Returns` block. I think it'd be helpful for our future selves to quickly be able to tell that you expect a list of 2 edge positions in pixels (I'm assuming just from reading the code)

Reply:
Yeah, good point.